test/functional/basic/basic-e2e-specs.js

```diff
@@ -92,6 +88,17 @@ describe('XCUITestDriver - basics -', function () {
       actual.viewportRect.width.should.be.a('number');
       delete actual.viewportRect;
 
+      // convert w3c caps into mjswp caps
+      let mjswpCaps = {};
```

Comment:
there should be no occurrences of deprecated capabilities. We should only use w3c stuff

Reply:
actually, getSession API itself is not a w3c one 🤷

Reply:
right, but I couldn't find a way to obtain w3c capabilities from appium server. I can continue working on this tests if you provide an alternative approach.

Reply:
maybe we don't need this test at all...
Let's keep it as is for now and leave a TODO there

Reply:
I see, thanks. BTW, it is ok to use mjsonwp and appium (`/session/:sessionId/appium/`) APIs?

Reply:
Eventually I would love to deprecate all non-w3c endpoints and move them under `mobile:`-like extensions, so the `external` REST spec is 100% compatible with webdriver standard.